compositor_render/src/transformations/shader/node.rs

```diff
@@ -33,6 +34,7 @@ impl ShaderNode {
             _custom_params_buffer: custom_params_buffer,
             shader,
             resolution: *resolution,
+            wgpu_ctx: ctx.wgpu_ctx.clone(),
```

Comment:
Why are you adding it? ctx is passed to the render method, you don't need to store it in node.

Reply:
I first added it when prototyping and forgot to remove it later. Good catch